website/docs/examples/multi_client_geoimage/README.md

```diff
@@ -0,0 +1,236 @@
+---
+id: multi-client-geoimage
+title: Multi Client GeoImage
+---
+
+### [Link to `multi_client_geoimage/main.py`](https://github.com/farm-ng/farm-ng-amiga/blob/main/py/examples/multi_client_geoimage/main.py)
```

Comment:
```suggestion
### [Link to `multi_client_geoimage/main.py`](https://github.com/farm-ng/farm-ng-amiga/blob/main-v2/py/examples/multi_client_geoimage/main.py)
```

Reply:
i was expecting that we merge merge that branch soon